app/templates/partials/answer.html

```diff
@@ -1,7 +1,8 @@
 {% import 'macros/helpers.html' as helpers %}
 
-<div class="answer answer--{{answer.schema_item.type|lower}}" id="{{answer.id}}">
-  {% set invalid = answer.is_valid == False %}
+{% set invalid = answer.is_valid == False %}
+
+<div class="answer answer--{{answer.schema_item.type|lower}} {{'js-has-errors' if invalid}}" id="{{answer.id}}">
```

Comment:
Instead of `if invalid ` can we not do something like `if not answer.is_valid`?
Im not sure if thats possible